<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-ICBR68V" name="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance" tests="7" failures="0" timestamp="05 Mar 2018 11:01:21 GMT" time="61.515" errors="1">
  <testcase name="searchEmployeeRecord" time="40.724" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
  <testcase name="clickOnSearchedEmployeeAndRewardTab" time="14.921" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
  <testcase name="verifyRegularPaymentAmount" time="0.159" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
  <testcase name="verifyOtherPaymentDeductionAndClicked" time="5.087" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
  <testcase name="verifyAnnualBenefit" time="0.242" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
  <testcase name="afterMethod" time="0.000" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance">
    <error type="org.openqa.selenium.NoSuchSessionException" message="The FirefoxDriver cannot be used after quit() was called.
Build info: version: &#039;3.0.1&#039;, revision: &#039;1969d75&#039;, time: &#039;2016-10-18 09:49:13 -0700&#039;
System info: host: &#039;DESKTOP-ICBR68V&#039;, ip: &#039;192.168.1.130&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_151&#039;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: The FirefoxDriver cannot be used after quit() was called.
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'DESKTOP-ICBR68V', ip: '192.168.1.130', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:464)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:658)
at org.openqa.selenium.firefox.FirefoxDriver.getScreenshotAs(FirefoxDriver.java:428)
at com.test.xcdhr.Core_HR_Sence_Check.testBase.TestBase.captureScreen(TestBase.java:141)
at com.test.xcdhr.Core_HR_Sence_Check.testBase.TestBase.getresult(TestBase.java:157)
at com.test.xcdhr.Core_HR_Sence_Check.testBase.TestBase.afterMethod(TestBase.java:172)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterMethod -->
  <testcase name="closeBroswe" time="0.382" classname="com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance"/>
</testsuite> <!-- com.test.xcdhr.Core_HR_Sence_Check.AddOherPaymentOrDeduction.TC001_VerifyOtherPaymentOrDeductionBenefitTypeAllowance -->
